README.md

```diff
@@ -1,10 +1,29 @@
-# Motivation
+# iRODS Capability - Indexing
+
+## Motivation
 
 The iRODS indexing capability provides a policy framework around both full text and metadata indexing for the purposes of enhanced data discovery.  Logical collections are annotated with metadata which indicates that any data objects or nested collections of data object should be indexed given a particular indexing technology, index type and index name.
 
-# Configuration
+**IMPORTANT: This project supports elasticsearch v7.0.0 and later.**
```

Comment:
do we feel comfortable (yet) saying we support opensearch as well?   i guess we need it in testing before we do.

Reply:
We need to test it to be sure. I don't want to state that just yet.

Reply:
agreed

Reply:
OpenSearch has docker containers so we can test it.

Personally, I think the plugin will work with OpenSearch, but assuming isn't the right way to handle it.

I think I'm going to look into moving the tests into a docker compose setup so it's easy to test the different indexing technologies.